cpg_pipes/targets.py

```diff
@@ -0,0 +1,639 @@
+"""
+Pipeline stage target and subclasses: Cohort, Dataset, Sample.
```

Comment:
At the moment this also includes pedigree classes. Should those be moved to a separate module?

Reply:
Thought about it a bit. We indeed don't need PedigreeInfo at all. Created an issue:
https://github.com/populationgenomics/production-pipelines/issues/39